### DISCUSSION TITLE
Standalone usage of bs-isotope

### DISCUSSION THREAD

**Discussion by e-orlov** [bootscore/bootscore#429] [General]
Hey!
Is it possible to use bs-isotope plugin standalone, without bootScore theme?

**Comment by crftwrk** [comment]
Hello,

not that simple, because plugin is designed for bootScore theme and uses functions from it. These functions have to be removed from plugin, otherwise you get a php error. If functions are removed it should work with other themes too, but you need a theme which has Bootstrap 5 included.

There are a lot (standalone) isotope plugins out there https://wordpress.org/plugins/tags/isotope/. Maybe one of this fit to your needs?

**Comment by e-orlov** [comment]
I plan to use LiveCanvas, https://livecanvas.com/ and Picostrap, https://picostrap.com/. Picostrap is based on Bootstrap5. Could you maybe take a short look into it, to say about isotope usage with it?

And, I think, to make bs-isotope usable as standalone, specially for WooCommerce, is not a bad idea.

greets
evgeniy


________________________________
Von: Basti ***@***.***>
Gesendet: Montag, 20. März 2023 15:36
An: bootscore/bootscore ***@***.***>
Cc: Evgeniy ***@***.***>; Author ***@***.***>
Betreff: Re: [bootscore/bootscore] Standalone usage of bs-isotope (Discussion #429)


Hello,

not that simple, because plugin is designed for bootScore theme and uses functions from it. These functions have to be removed from plugin, otherwise you get a php error. If functions are removed it should work with other themes too, but you need a theme which has Bootstrap 5 included.

There are a lot (standalone) isotope plugins out there https://wordpress.org/plugins/tags/isotope/. Maybe one of this fit to your needs?

**Reply by crftwrk** [reply]
You can use LiveCanvas with bootScore as well, no need to use Picostrap. 

But yes, Picostrap is a good theme and based on Bootstrap 5. When functions are removed as described in previous comment, it should work. I can you help out here if you want.
